auditors/deprecatedapis/depreceatedapis_test.go

```diff
@@ -53,6 +54,12 @@ func TestAuditDeprecatedAPIs(t *testing.T) {
 			require.Nil(t, err)
 			report := test.AuditManifest(t, fixtureDir, tc.file, auditor, []string{DeprecatedAPIUsed})
 			assertReport(t, report, tc.expectedSeverity, message, metadata)
+
+			// disable local tests when running in dev mode
+			if os.Getenv("USE_KIND") == "false" {
+				return
+			}
+
```

Comment:
had to add this in order to run tests locally

Reply:
that seems fine here